src/clib/cool1d_multi_g.F

```diff
@@ -1015,7 +1015,7 @@ subroutine cool1d_multi_g(
                 grbeta = 0.74_DKIND / tgas(i)**0.068_DKIND
                 edot(i) = edot(i) -
      &               regr(i) * (myisrf(i)*dom_inv / myde(i))**grbeta *
-     &               myde(i) * rhoH(i)
```

Comment:
Is this a bug fix? If yes, can you mention and describe in the PR description.

Reply:
I believe it is. Thank you for pointing out, that I missed mentioning it in the PR description. I think conceptually it should be clear, why the recombination cooling on grains should be proportional to the normalization of the grain size distribution. Wolfire+ 1995 adopted this expression from Bakes & Tielens 1994, who assumed the MRN grain size distribution.

Reply:
I agree, although I think you mean proportional to the dust to gas ratio. I added these factors of `dust2gas(i) / fgr` in a couple places when I added the ability to provide a dust density field, but I must have missed this one.